security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/SecurityAnalysisParametersSerializer.java

```diff
@@ -30,12 +30,15 @@ public void serialize(SecurityAnalysisParameters parameters, JsonGenerator jsonG
         jsonGenerator.writeStartObject();
 
         jsonGenerator.writeStringField("version", SecurityAnalysisParameters.VERSION);
-
+        jsonGenerator.writeStringField("increasedFlowViolationsThreshold", String.valueOf(parameters.getIncreasedFlowViolationsThreshold()));
```

Comment:
Since "loadflow-parameters" is in snake case, we should probably use snake case for all parameters (although different from fields inside the loadflow parameters).

Reply:
We have not been consistent in existing parameters naming, I don't know which one is better to follow.

Reply:
I think we should be consistent at least within a class : here loadflow parameters is already in snake case!
Or indeed we could change it, in order to be consistent with the fields of loadflow parameters?

Reply:
Ok let's change these new params to camel case

Reply:
Done